FPIS/Views/CreateWaterForm.cs

```diff
@@ -0,0 +1,121 @@
+﻿using FPIS.Models;
+using FPIS.Services;
+using FPIS.Utils;
+using MaterialSkin.Controls;
+using System.Diagnostics.Metrics;
+
+namespace FPIS.Views
+{
+    public partial class CreateWaterForm : MaterialForm
+    {
+        private bool _isDataValid = true;
+
+        public CreateWaterForm()
+        {
+            InitializeComponent();
+            materialLabelWaterNameError.ForeColor = System.Drawing.Color.Red;
+        }
+        public void ClearFormFields()
+        {
+            materialTextBoxWaterName.Text = "";
+        }
+
+        public void ClearErrorLabels()
+        {
+            materialLabelWaterNameError.Text = "";
+        }
+
+        public void ValidateWaterName()
+        {
+            string waterName = materialTextBoxWaterName.Text.Trim();
+
+            if (waterName.Length == 0)
+            {
+                this._isDataValid = false;
+                materialLabelWaterNameError.Text = "Water name is required!";
+            }
+        }

```

Comment:
Kindly add a validation to avoid numbers and special characters

Reply:
Validation is added